crates/typst/src/eval/mod.rs

```diff
@@ -21,8 +21,10 @@ mod methods;
 mod module;
 mod none;
 pub mod ops;
+mod plugin;
 mod scope;
 mod symbol;
+pub use plugin::{Plugin, PluginInstance};
```

Comment:
could you add `self::` before `plugin` and move it to the other reexports below?